packages/playground/Samples/accessible.tsx

```diff
@@ -50,7 +61,34 @@ export default class Bootstrap extends React.Component<
           }}>
           <Text style={styles.text}>{this.state.displayText}</Text>
         </View>
-        <TextInput />
+        <TouchableHighlight
+          style={styles.item}
+          onPress={() => {
+            AccessibilityInfo.announceForAccessibility('Testing Testing 1 2 3');
+          }}
+          {...{
+            // Use weird format as work around for the fact that these props are not part of the @types/react-native yet
+            focusable: true,
+            accessibilityLabel: 'TEST Announce For Accessibility',
+          }}>
```

Comment:
```suggestion
          }}
          accessibilityLabel="TEST Announce For Accessibility"
          {...{
            // Use weird format as work around for the fact that this prop is not part of the @types/react-native yet
            focusable: true,
          }}>
```
I think you can use accessibilityLabel in normal format now, maybe something like this would make it a bit clearer. Same thing in the focus example.

Reply:
Sure I'll check that out to see if the types have been fixed.